src/test/java/cloud/fogbow/ras/core/plugins/interoperability/openstack/securityrule/v2/OpenStackSecurityRulesPluginTest.java

```diff
@@ -43,474 +51,844 @@
 })
 public class OpenStackSecurityRulesPluginTest extends BaseUnitTests {
 
-    private static final int FAKE_PORT_FROM = 1024;
-    private static final int FAKE_PORT_TO = 2048;
-    
-    private static final String ANY_STRING = "any-string";
-    private static final String ANY_URL = "http://localhost:8007";
-    private static final String NETWORK_NEUTRONV2_URL_KEY = "openstack_neutron_v2_url";
-    private static final String DEFAULT_NETWORK_URL = "http://localhost:0000";
-    private static final String INGRESS_TRAFFIC = "ingress";
-    private static final String SECURITY_RULE_ID = "securityRuleId";
-    private static final String SECURITY_GROUP_ID = "securityGroupId";
+    private static final String NETWORK_PREFIX_ENDPOINT = "https://mycloud.domain:9696";
+    private static final String QUERY_SECURITY_GROUP_ID = "?security_group_id=";
```

Comment:
There are ways more elegant to create this "query" strings and JSON but I will not ask to change it now because we need to review all plugins to do it.

Reply:
Certainly, I really liked the way you did the tests on CloudStack, but here it was not practical to perform this kind of test refactoring on this task.